src/styles/blocks/_catalog.scss

```
@@ -14,9 +14,16 @@
     border: 1px solid #ccc;
     border-radius: 5px;
     margin-bottom: 48px;
+
+    transition: transform, color 0.3s;
```

You need to set duration for all rules.
```suggestion
    transition: transform 0.3s, color 0.3s;
```